version.c

```diff
@@ -505,6 +505,7 @@ void print_version(FILE *fp)
   rstrip_in_place((char *) configure_options);
   fprintf(fp, "\n\nConfigure options: %s\n", (char *) configure_options);
 
+#ifndef DISABLE_WRITING_CFLAGS
   rstrip_in_place((char *) cc_cflags);
   fprintf(fp, "\nCompilation CFLAGS: %s\n", (char *) cc_cflags);
```

Comment:
I'd terminate the ifndef block here. Do "default options" and "compile options" also cause problems to guix/nix?

Reply:
You are correct, it's only "Compilation CFLAGS" that matters for us. I've fixed this.